controllers/monitoring_alert.go

```diff
@@ -271,6 +270,7 @@ func (r *ManagedMCGReconciler) configReceiver(dmsURL string, alertingAddressList
 				receiver.EmailConfigs[0].To = strings.Join(alertingAddressList, ", ")
 				receiver.EmailConfigs[0].HTML = string(smtpHTML)
 			} else {
+				receiver.EmailConfigs = []promv1a1.EmailConfig{}
```

Comment:
do we need to initialize EmailConfigs here?

Reply:
This was here originally, PTAL https://github.com/red-hat-storage/mcg-osd-deployer/pull/18/files#diff-a53791a04f3985464c97e8af8a4c8e7407c0cd6397f0dd855cd1b60a340b156fL268.

Reply:
@rexagod Please check the following code is making use of that code, if not please remove it

Reply:
Removed and verified. Reconciliation happens without any errors.

![Screenshot from 2022-05-04 17-12-56](https://user-images.githubusercontent.com/33557095/166674824-454377fe-238f-4b5b-9aed-8f219590010f.png)